apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-recurring-payments.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_close_click`.
+ *
+ * @returns {{handler: Function, selector: string, type: string}} event object definition.
+ */
+export default () => ( {
+	selector: '.wp-block[data-type="jetpack/recurring-payments"] .components-button',
```

Comment:
Maybe `.wp-block[data-type="jetpack/recurring-payments"] .premium-content-block-nudge__button`

Since we link to WordPress.com/checkout it's likely also possible to pass through a query param and handle it on the Calypso side.